### PR TITLE
Fix sonar issues

### PR DESCRIPTION
**Pull request by sdimitrov9**
**Description**:

Fix sonar issues. Most of the issues left outside of the PR are related to web3 copied services code.

**Related issue(s)**:

Related to #9010

**Notes for reviewer**:

Test `wipesSerialsWhenApprovedForAll()` is now deleted since it duplicates another test's content since its creation. Several 4+ years old TODOs are removed from tests.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/UpdateTokenFreezeKycStatusMigrationTest.java [hunk 1]
```diff
@@ -154,6 +154,9 @@ void migrate() {
         assertThat(findHistory("token", PostMigrationToken.class)).containsExactlyInAnyOrderElementsOf(expectedHistory);
     }
 
+    @SuppressWarnings(
+            "java:S1854") // timestamp variable is necessary in order to avoid repetition and to make code easier to
+    // read
```
Comment (steven-sheehy) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/UpdateTokenFreezeKycStatusMigrationTest.java:159]: nit: Shorten comment so it and annotation appear on one line

File: hedera-mirror-web3/src/main/java/com/hedera/services/fees/usage/token/entities/TokenEntitySizes.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 /**
  *  Exact copy from hedera-services
  */
+@SuppressWarnings("squid:S6548") // enum soon to be deleted with the introduction of reusable services
```
Comment (steven-sheehy) [inline hedera-mirror-web3/src/main/java/com/hedera/services/fees/usage/token/entities/TokenEntitySizes.java:28]: Is this still needed with sonar-project.properties?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/provider/AbstractStreamFileProviderTest.java [hunk 3]
```diff
@@ -56,6 +56,7 @@ abstract class AbstractStreamFileProviderTest {
     protected StreamFileProvider streamFileProvider;
 
     @BeforeEach
+    @SuppressWarnings("java:S1130") // Exception is thrown in the child classes setup method
```
[the remark this answers is not in the document]
Comment (sdimitrov9) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/provider/AbstractStreamFileProviderTest.java]: Done

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/provider/AbstractStreamFileProviderTest.java [hunk 4]
```diff
@@ -71,6 +72,7 @@ void setup() throws Exception {
 
     protected abstract String resolveProviderRelativePath(ConsensusNode node, String fileName);
 
+    @SuppressWarnings("java:S1172") // node is used in the child classes implementations
```
[the remark this answers is not in the document]
Comment (sdimitrov9) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/provider/AbstractStreamFileProviderTest.java]: Done

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/UpdateTokenFreezeKycStatusMigrationTest.java (hunk 1 above)
Comment (sdimitrov9) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/UpdateTokenFreezeKycStatusMigrationTest.java]: Done

File: hedera-mirror-web3/src/main/java/com/hedera/services/fees/usage/token/entities/TokenEntitySizes.java (hunk 2 above)
Comment (sdimitrov9) [reply hedera-mirror-web3/src/main/java/com/hedera/services/fees/usage/token/entities/TokenEntitySizes.java]: Shouldn't be, it is removed now.

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**